portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -3162,6 +3162,53 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-126479."
+	@priority = "4"
+	test SubmitFormWhenSessionExpires {
+		property portal.acceptance = "false";
+		property portal.upstream = "true";
+		property test.name.skip.portal.instance = "Forms#SubmitFormWhenSessionExpires";
+		property web.xml.timeout = "2";
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.addTextField(fieldPositionNumber = "1");
+
+		var textFieldName = FormFields.getFieldName();
+
+		Form.closeSidebar();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormFields.editText(
+			fieldName = "${textFieldName}",
+			fieldValue = "Test");
+
+		Pause(locator1 = "120000");
```

Comment:
@carolmariaabb, for sure, Brian will complain about it, please add a comment above this line to explain its need, e.g. the custom time to the session expire and the test succeed.